cg/store/api/findbasicdata.py

```diff
@@ -68,6 +68,10 @@ def customer_by_id(self, id_: int) -> models.Customer:
         """Fetch a customer by id number from the store."""
         return self.Customer.query.filter_by(id=id_).first()
 
+    def sample_by_id(self, id_: int) -> models.Sample:
```

Comment:
```suggestion
    def get_sample_by_id(self, sample_id: str) -> Sample:
```

Reply:
This function fetches the sample by its entry id, which is an int. So I think the type should remain as it is.